### PR TITLE
Feature/product gallery

### PR DESCRIPTION
**Pull request by luisarmandodv**
This corresponds to:
https://github.com/magma-labs/shopify-themes/issues/33
https://github.com/magma-labs/shopify-themes/issues/34
https://github.com/magma-labs/shopify-themes/issues/35
https://github.com/magma-labs/shopify-themes/issues/36
https://github.com/magma-labs/shopify-themes/issues/37
https://github.com/magma-labs/shopify-themes/issues/38
https://github.com/magma-labs/shopify-themes/issues/39

### CODE REVIEW

File: MH-FiorellaTheme5/source/partials/_info-product.erb [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<section class="product">
+  <div class="product-container-mobile">
+    <h2 class="product-title">Flower Bouquet Victoria</h2>
+    <img/>
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/partials/_info-product.erb]: You have an image tag here without a source, why is that?

File: MH-FiorellaTheme5/source/partials/_info-product.erb [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        <option value="5">5</option>
+        <option value="6">6</option>
+      </select>
+      <i class="fa fa-caret-down" aria-hidden="true"></i>
+    </div>
+    <p class="product-price">$84.50</p>
+    <button class="product-button-cart">ADD TO CART</button>
+    <h3 class="title-footer">Share:</h3>
+    <div class="share-icons">
+      <div><img/></div>
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/partials/_info-product.erb]: If you are going to insert images, add a source. If not then use a `div` and a `background-image:`

File: MH-FiorellaTheme5/source/partials/_info-product.erb [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      <div><img/></div>
+      <div><img/></div>
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/partials/_info-product.erb]: Also here

File: MH-FiorellaTheme5/source/partials/_info-product.erb [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    </div>
+  </div>
+  <div class="product-container-desktop">
+    <div class="product-img">
+      <img id="mainImage" src="/images/pdp/bouquet_victoria_pdp_desktop.jpg"/>
+      <div class="product-img-thumbnail">
+        <div class="image-container">
+          <div>
+            <img src="/images/pdp/pdp_thumbnail_1.jpg" class="thumbnail"/>
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/partials/_info-product.erb]: This is done right, why aren't you doing the same for the other images?

File: MH-FiorellaTheme5/source/partials/_info-product.erb [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+          <option value="6">6</option>
+        </select>
+        <i class="fa fa-caret-down" aria-hidden="true"></i>
+      </div>
+      <p class="product-price">$84.50</p>
+      <button class="product-button-cart">ADD TO CART</button>
+      <div class="share-container">
+        <b class="share-title">Share:</b>
+        <div class="share-icons">
+          <div><img/></div>
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/partials/_info-product.erb]: Add source along with the 2 below this.

File: MH-FiorellaTheme5/source/stylesheets/pdp-styles.css [hunk 6]
```diff
@@ -0,0 +1,274 @@
...
+}
+
+.product .colors-bouquet span {
+  border-radius: 25px;
+  display: inline-block;
+  height: 50px;
+  width: 50px;
+}
+
+.product .colors-bouquet span:nth-child(1) {
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/pdp-styles.css]: Hmmm not a big fan of specifying children. I would recommend to add a class to each and one of the `span` and add the styles from there.

File: MH-FiorellaTheme5/source/partials/_info-product.erb (hunk 1 above)
Comment (luisarmandodv) [reply MH-FiorellaTheme5/source/partials/_info-product.erb]: I'm adding it on styles.css... should I change it?

File: MH-FiorellaTheme5/source/pdp.html.erb [hunk 7]
```diff
@@ -0,0 +1,9 @@
+<div class="container">
+  <%= partial "partials/slide-out-menu" %>
+
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/pdp.html.erb]: Why you have this blank lines, what do you think about remove it?

File: MH-FiorellaTheme5/source/stylesheets/pdp-styles.css [hunk 8]
```diff
@@ -0,0 +1,258 @@
...
+
+.product .colors-bouquet span {
+  border-radius: 25px;
+  display: inline-block;
+  height: 50px;
+  width: 50px;
+}
+
+.product .colors-bouquet .pink-circle {
+  background-color: pink;
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/pdp-styles.css]: Please use the Hex value.

File: MH-FiorellaTheme5/source/stylesheets/pdp-styles.css [hunk 9]
```diff
@@ -0,0 +1,258 @@
...
+  margin-right: 20px;
+}
+
+.product .colors-bouquet .orange-circle {
+  background-color: orange;
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/pdp-styles.css]: Please use the Hex value.

File: MH-FiorellaTheme5/source/stylesheets/pdp-styles.css [hunk 10]
```diff
@@ -0,0 +1,258 @@
...
+  margin-right: 20px;
+}
+
+.product .colors-bouquet .red-circle {
+  background-color: red;
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/pdp-styles.css]: Please use the Hex value.

File: MH-FiorellaTheme5/source/stylesheets/pdp-styles.css [hunk 11]
```diff
@@ -0,0 +1,258 @@
...
+.product .product-price {
+  align-self: flex-start;
+  color: #E83671;
+  font-family: "MerriweatherSans-Bold";
+  font-size: 30px;
+}
+
+.product button {
+  background-color: #E83671;
+  color: white;
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/pdp-styles.css]: Please use the Hex value.

File: MH-FiorellaTheme5/source/stylesheets/pdp-styles.css [hunk 12]
```diff
@@ -0,0 +1,258 @@
...
+  }
+
+  .product .product-container-desktop .product-img .product-img-thumbnail .image-container {
+    max-height: 54px;
+    max-width: 54px;
+    padding: 4px;
+  }
+
+  .product .product-container-desktop .product-img .product-img-thumbnail .image-container:hover {
+    border: 1px solid red;
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/pdp-styles.css]: Please use the Hex value.

**Pull request merged**